traveler-swift-core/TravelerKit/Models/ParkingItem.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  ParkingItem.swift
+//  TravelerKit
+//
+//  Created by Omar Padierna on 2019-10-08.
+//  Copyright © 2019 Ata Namvari. All rights reserved.
+//
+
+import Foundation
+
+public struct ParkingItem: ProductItem, Decodable, Product {
```

Comment:
lets see the effort required to remove `Product`

Reply:
too much effort. ProductItem inherits from CatalogItem. Product does not inherit from anybody. If BookingProduct (or any other product for that matter) would conform to ProductItem it would require to implement attributes it doesnt have like a thumbnail or a subtitle.

Reply:
Product stays, ProductItem has been removed. ParkingItem and BookingItem now conform to CatalogItem and to Product.